### DISCUSSION TITLE
Share Tmog unlocks of BoPs in group/raids

### DISCUSSION THREAD

**Discussion by Tehmirror** [chromiecraft/chromiecraft#6349] [Suggestions]
Currently you often have the situation in dungeon groups and raids that people are trading BoP items back and fourth because of transmog unlocks. This does not have a downside for those doing the trades but slows the group a bit and sometimes annoys a few players. Sometimes you even see "tmog need" in 5man groups.

I want to suggest, that transmogs for BoP items are shared with everyone in the group/raid as soon as they are distributed for convenience reasons.

**Comment by Plan414** [comment]
This would be nice, especially when players DE items before we can even type in chat.
